uint8 motor_id
int8 speed
---
bool done